shell/browser/api/electron_api_session.cc

```diff
@@ -1043,6 +1043,7 @@ base::Value Session::GetSpellCheckerLanguages() {
 void Session::SetSpellCheckerLanguages(
     gin_helper::ErrorThrower thrower,
     const std::vector<std::string>& languages) {
+#if !BUILDFLAG(IS_MAC)
```

Comment:
_(pedantic)_ arguably a `semver/minor` change since this an invalid URL no longer throws on macOS?

Reply:
same as above